src/Collections-Strings-Tests/StringTest.class.st

```diff
@@ -577,20 +577,6 @@ StringTest >> testBeginsWithCaseSensitive [
 	self assert: (w1 beginsWith: w2 asUppercase caseSensitive: false).
 ]
 
-{ #category : #tests }
-StringTest >> testBeginsWithEmptyCaseSensitive [
-
-	self deny: ('éèàôüößäóñíá' beginsWithEmpty: 'ß' caseSensitive: true).
```

Comment:
I found many other StringTests with strange encodings. Not sure what to do with them.  But such cleaning required a separate issue